supertokens_python/recipe/emailpassword/types.py

```diff
@@ -84,3 +87,10 @@ def __init__(
     TypeEmailPasswordPasswordResetEmailDeliveryInput,
     TypeEmailVerificationEmailDeliveryInput
```

Comment:
Similar to `TypeEmailVerificationEmailDeliveryInput`, you don't require user_context in `TypeEmailPasswordPasswordResetEmailDeliveryInput`

Reply:
@bhumilsarvaiya If I do that, https://github.com/supertokens/supertokens-python/pull/102/files#diff-074864472d77ea0c5ef0cd85a5b5bedddd0c510b6c7a0aa70cafd20017331ad0R131 starts saying that it needs UserContext. The `user_context` in `get_user_by_id` isn't getting used but it's present. So need to pass something. 

We can pass `{}` but won't that be dangerous?

Reply:
we already have `user_context` passed as parameter for `send_email`. Pass that to `get_user_by_id` instead